test/extended/util/annotate/rules.go

```diff
@@ -204,6 +204,8 @@ var (
 			`\[sig-network\] Services should be able to switch session affinity for service with type clusterIP`,
 			`\[sig-network\] Services should have session affinity work for NodePort service`,
 			`\[sig-network\] Services should have session affinity work for service with type clusterIP`,
+			`\[sig-network\] Services should have session affinity timeout work for NodePort service`,
```

Comment:
Please link these comments to a bug or card that we expect to fix them

Reply:
@smarterclayton They already are: the same card as the section they are in, SDN-510. They are new tests added recently for affinity, and we already have a card for that.